internal/testing/database/postgres.go

```diff
@@ -25,8 +29,11 @@ type TestingT interface {
 func PostgresDriver(t TestingT, schemaSuffix string) *Driver {
 	t.Helper()
 	pgConn, ok := os.LookupEnv("POSTGRESQL_CONNECTION")
-	if !ok {
-		return nil
+	switch {
+	case !ok && isEnvironmentCI:
+		t.Fatal("CI must test all drivers, set POSTGRESQL_CONNECTION")
+	case !ok:
+		t.Skip("Set POSTGRESQL_CONNECTION to test against postgresql")
```

Comment:
So many of our tests rely on this I wonder if we should always fail without it

Reply:
The tests for our main interfaces (CLI, API, config) do all rely on a database, but there are plenty of tests for internal things that do not. It's easy to change, but I think it can be nice to let people run tests with minimal setup.